esp8266-fastled-webserver/Fields.cpp

```diff
@@ -274,31 +420,46 @@ FieldList fields = {
+    {"brightDepthBpm", "Brightness Depth BPM",    NumberFieldType, 0, 255, getBrightDepthBpm, nullptr, setBrightDepthBpm},
+    {"brightDepthMin", "Brightness Depth Min",    NumberFieldType, 0, 255, getBrightDepthMin, nullptr, setBrightDepthMin},
+    {"brightDepthMax", "Brightness Depth Max",    NumberFieldType, 0, 255, getBrightDepthMax, nullptr, setBrightDepthMax},
 
     {"brightThetaIncBpm", "Bright Theta Inc BPM", NumberFieldType, 0, 255, getBrightThetaIncBpm, nullptr, setBrightThetaIncBpm},
     {"brightThetaIncMin", "Bright Theta Inc Min", NumberFieldType, 0, 255, getBrightThetaIncMin, nullptr, setBrightThetaIncMin},
     {"brightThetaIncMax", "Bright Theta Inc Max", NumberFieldType, 0, 255, getBrightThetaIncMax, nullptr, setBrightThetaIncMax},
 
-    {"msMultiplierBpm", "Time Multiplier BPM", NumberFieldType, 0, 255, getMsMultiplierBpm, nullptr, setMsMultiplierBpm},
-    {"msMultiplierMin", "Time Multiplier Min", NumberFieldType, 0, 255, getMsMultiplierMin, nullptr, setMsMultiplierMin},
-    {"msMultiplierMax", "Time Multiplier Max", NumberFieldType, 0, 255, getMsMultiplierMax, nullptr, setMsMultiplierMax},
+    {"msMultiplierBpm", "Time Multiplier BPM",    NumberFieldType, 0, 255, getMsMultiplierBpm, nullptr, setMsMultiplierBpm},
+    {"msMultiplierMin", "Time Multiplier Min",    NumberFieldType, 0, 255, getMsMultiplierMin, nullptr, setMsMultiplierMin},
+    {"msMultiplierMax", "Time Multiplier Max",    NumberFieldType, 0, 255, getMsMultiplierMax, nullptr, setMsMultiplierMax},
 
-    {"hueIncBpm", "Hue Inc BPM", NumberFieldType, 0, 255, getHueIncBpm, nullptr, setHueIncBpm},
-    {"hueIncMin", "Hue Inc Min", NumberFieldType, 0, 255, getHueIncMin, nullptr, setHueIncMin},
-    {"hueIncMax", "Hue Inc Max", NumberFieldType, 0, 255, getHueIncMax, nullptr, setHueIncMax},
+    {"hueIncBpm", "Hue Inc BPM",                  NumberFieldType, 0, 255, getHueIncBpm, nullptr, setHueIncBpm},
+    {"hueIncMin", "Hue Inc Min",                  NumberFieldType, 0, 255, getHueIncMin, nullptr, setHueIncMin},
+    {"hueIncMax", "Hue Inc Max",                  NumberFieldType, 0, 255, getHueIncMax, nullptr, setHueIncMax},
 
-    {"sHueBpm", "S Hue BPM", NumberFieldType, 0, 255, getSHueBpm, nullptr, setSHueBpm},
-    {"sHueMin", "S Hue Min", NumberFieldType, 0, 255, getSHueMin, nullptr, setSHueMin},
-    {"sHueMax", "S Hue Max", NumberFieldType, 0, 255, getSHueMax, nullptr, setSHueMax},
+    {"sHueBpm", "S Hue BPM",                      NumberFieldType, 0, 255, getSHueBpm, nullptr, setSHueBpm},
+    {"sHueMin", "S Hue Min",                      NumberFieldType, 0, 255, getSHueMin, nullptr, setSHueMin},
+    {"sHueMax", "S Hue Max",                      NumberFieldType, 0, 255, getSHueMax, nullptr, setSHueMax},
 };
 
-uint8_t fieldCount = ARRAY_SIZE(fields);
+const uint8_t fieldCount = ARRAY_SIZE2(fields);
+// TODO: consider using ArduinoJSON ... pre-allocated buffer, simpler usage, tested code
+
+Field getField(String name) {
```

Comment:
SIMPLE FILE REVIEW

The following are the only externally-used functions, and are therefore defined in the common header files.